specification/_global/field_caps/FieldCapabilitiesRequest.ts

```diff
@@ -75,6 +75,12 @@ export interface Request extends RequestBase {
      * @availability serverless
      */
     types?: string[]
+    /**
+     * If false, empty fields are not included in the response.
+     * @availability stack since=8.13.0
```

Comment:
I haven't seen anything that restrict this to `stack` only.
Do we miss a `serverless` availability as well ?

Reply:
You are right, adding it! Thanks